_data/sites.json

```diff
@@ -1321,6 +1321,16 @@
         ]
     },
 
+    {
+        "name": "BeyondMenu",
+        "url": "https://www.beyondmenu.com/manageaccount.aspx",
+        "difficulty": "easy",
+        "notes": " Sites with a simple process such as a 'delete account' button.",
```

Comment:
```suggestion
```

This is just the description of what an easy entry is, it seems to be simple enough, so we can leave it without anything and use the default note.

Reply:
Hello, Updated as per notes above :)